<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IvWCSDockWidget</class>
 <widget class="QDockWidget" name="IvWCSDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>CD1_1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="cd11LineEdit"/>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QSlider" name="cd11Slider">
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>CD1_2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="cd12LineEdit"/>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QSlider" name="cd12Slider">
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>CD2_1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="cd21LineEdit"/>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QSlider" name="cd21Slider">
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>CD2_2</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="cd22LineEdit"/>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QSlider" name="cd22Slider">
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Plate scale</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLineEdit" name="plateScaleLineEdit">
      <property name="statusTip">
       <string>The plate scale in arcsec / pixel</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="QSlider" name="plateScaleSlider">
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Pos. angle</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLineEdit" name="posAngleLineEdit">
      <property name="statusTip">
       <string>Position angle, counting North over East</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="QSlider" name="posAngleSlider">
      <property name="minimum">
       <number>-100</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="3">
     <widget class="QPushButton" name="resetPushButton">
      <property name="text">
       <string>Restore original WCS</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
